composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -129,8 +146,8 @@ fun parseArgs(rawArgs: Array<String>): Args {
                     }
                 },
                 verboseOutput = jCommanderArgs.verboseOutput ?: false,
-                devices = jCommanderArgs.devices ?: emptyList()
+                devices = jCommanderArgs.devices ?: emptyList(),
+                devicePattern = jCommanderArgs.devicePattern ?: ""
         )
-    }
+    }.let { args -> validateArguments(args) }
```

Comment:
`args` was unresolved reference error in this case.
What about define `validateArguments` as a member function of `Args`?
(I've defined it as a member function in aa4688a)